Config.uk

```diff
@@ -0,0 +1,17 @@
+menuconfig LIBGEOS
+	bool "libgeos - a C/C++ library for GIS computational geometry"
+	default n
+	select LIBCXX
+	select CXX_THREADS
```

Comment:
Is `libcxx` the only library this depends on? If not, we should add the rest here too.